testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveViews.java

```diff
@@ -118,4 +118,22 @@ public void testLateralViewExplode()
                         row("nothing", null),
                         row("zero", null)));
     }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testLateralViewJsonTupleAs()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_json_tuple_table");
+        onTrino().executeQuery("" +
+                "CREATE TABLE test_json_tuple_table AS " +
+                "SELECT 3 id, CAST('{\"user_id\": 1000, \"first.name\": \"Mateusz\", \"Last Name\": \"Gajewski\", \".NET\": true}' AS varchar) jsonstr");
```

Comment:
@wendigo please review this query ;)

Reply:
it's ok but can we @findepi  add an non-existent field as well (which should yield NULL)

Reply:
Adding. And adding a field with json null value as well.